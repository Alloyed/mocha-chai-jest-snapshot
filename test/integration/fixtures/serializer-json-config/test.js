const { expect } = require("chai");

describe("foo", function () {
  it("bar", function () {
    expect("raw").toMatchSnapshot();
  });
});
