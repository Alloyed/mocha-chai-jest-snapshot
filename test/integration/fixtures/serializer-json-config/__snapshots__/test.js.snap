// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`foo bar 1`] = `serialized`;
